src/components/footer/index.njk

```diff
@@ -89,6 +90,12 @@ For badged services where ONS is providing services to external organisations, w
     patternlibExample({"path": 'components/footer/examples/footer-transactional-no-ogl-crest-nisra/index.njk'})
 }}
 
+#### Footer Warning
+
```

Comment:
Add lines: 

Use this if you need to show sensitive information on a post-submission page, such as the user's address or a response ID.  

The link should clear the any `sessionStorage` and send the user to a page outside of the transaction.